index.html

```diff
@@ -31,7 +31,7 @@
     <footer id="footer"><a>Click Here</a></footer>
 </body>
 </html>
-<!-- When you click a .nav-link button in the nav element, you are taken to the corresponding section of the landing page
+<!-- When ou click a .nav-link button in the nav element, you are taken to the corresponding section of the landing page
 You can watch an embedded product video with id="video"
 Your landing page has a form element with a corresponding id="form"
 Within the form, there is an input field with id="email" where you can enter an email address
```

Comment:
I think it is better to remove the comments from your code it makes the code looks good.